scripts/nginx/mylar.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Brett 2021
+function mylar_ngx() {
+    if [[ -f /install/.nginx.lock ]]; then
```

Comment:
This should be done in the installer. Check the structure of any other script in the nginx folder